PixelToolkit.py

```diff
@@ -11,13 +11,29 @@
 
     pass_gen = subparsers.add_parser("pass-gen", help="Password Generator")
     pass_gen.add_argument("--length", "-l", type=int, help="Length of the password", default=10)
+    
+    port_scan = subparsers.add_parser("port-scan", help="Port Scanner")
+    port_scan.add_argument("--host", type=str, help="Host to scan", default="127.0.0.1")
+    port_scan.add_argument("--ports", "-p", type=str, dest="port_range", default="1-65535", help="Port range to scan. Default is from 1 to 65535 (every port)")
```

Comment:
If I'm not mistaken you could access the ports value using args.ports instead of setting this dest. But that's also fine

Reply:
If that's fine I shall leave it for now.